tcn_hpl/data/utils/ptg_datagenerator.py

```diff
@@ -49,14 +66,9 @@ def create_training_data(config_path):
     top_k_objects = config["data_gen"]["top_k_objects"]
     pose_repeat_rate = config["data_gen"]["pose_repeat_rate"]
     exp_ext = config["data_gen"]["exp_ext"]
+    raw_data_root = f"{config['data_gen']['raw_data_root']}/{TASK_TO_NAME[task_name]}/Data"
+    dive_output_root = config["data_gen"]["dive_output_root"]
 
-    dset = kwcoco.CocoDataset(config["data_gen"]["dataset_kwcoco"])
-    # Check if the dest has activity gt, if it doesn't then add it
-    if not "activity_gt" in list(dset.imgs.values())[0].keys():
-        print("adding activity ground truth to the dataset")
-        from angel_system.data.common.kwcoco_utils import add_activity_gt_to_kwcoco
-
-        dset = add_activity_gt_to_kwcoco(topic, task_name, dset, activity_config_fn)
 
     def filter_dset_by_split(split):
```

Comment:
I'm looking at this due to the removal of the code just above. This function seems to require the parent-scope `dset` object. I see it being defined further below, however, I suggest making this requirement explicit in the function signature parameters (e.g. `def filter_dset_by_split(split, dataset):`) as opposed to implicit parent-scope pickup (and use a different name so as not to shadow the parent-scope symbol name).

Reply:
I agree - though this code was already part of the main branch prior to my changes. I can update it either way.